scripts/trace_flowcell.py

```diff
@@ -0,0 +1,118 @@
+import os
+import requests
+import json
+import sys
+import yaml
+import argparse
+from datetime import datetime
+
+
+
+
+def get_traces_for_tag(tag, headers):
+    traces_response = requests.get(
+        "{}/{}/?{}={}".format(api_base_url, "traces", "trace_tag", tag),
+        verify=False,
+        headers=headers)
+    if not traces_response.ok:
+        raise Exception("Response not OK, got: " + traces_response.text)
+
+    return json.loads(traces_response.text)
+
+def get_executions_from_traces(traces, headers):
+    for trace in traces:
+        trace_id = trace["id"]
+        trace_info_response = requests.get(
+            "{}/{}/{}".format(api_base_url, "traces", trace_id),
+            verify=False,
+            headers=headers)
+        trace_info = json.loads(trace_info_response.text)
+        action_executions = map(lambda x: x["object_id"], trace_info["action_executions"])
+        for execution in action_executions:
+            yield execution
+
+def get_actions_from_executions(executions, headers):
+    for execution_id in executions:
+        execution_info_response = requests.get(
+            "{}/{}/{}".format(api_base_url, "executions", execution_id),
+            verify=False,
+            headers=headers)
+        execution_info = json.loads(execution_info_response.text)
+        yield execution_info
+
+def filter_actions_by_name(actions, name):
+    return (action for action in actions if name in action["action"]["name"])
+
+def sort_actions_by_timestamp(actions):
+    def get_start_time(action):
+        #Select start_time, e.g. 2017-05-29T15:03:03.818222Z for each action. Remove everything after ".".
+        start_time = datetime.strptime(action['start_timestamp'].split(".")[0], "%Y-%m-%dT%H:%M:%S")
+        return start_time
+    #Compare elements in actions on key value start_timestamp.
+    sort_actions = sorted(actions, key=get_start_time)
+    return sort_actions
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description="Gets execution ids associated with a trace tag (e.g. a runfolder) and a "
+                                             "workflow. It can be used to track executions, e.g: "
+                                             "python scripts/trace_runfolder.py --tag 150605_M00485_0183_000000000-ABGT6_testbio14 | xargs -n1 st2 execution get")
+    parser.add_argument('--flowcell', required=True)
+    parser.add_argument('--api_base_url', default="https://localhost:4041/api/v1")
```

Comment:
Port number should be 9101 on `arteria-master`
```suggestion
    parser.add_argument('--api_base_url', default="https://localhost:9101/api/v1")
```

Reply:
actually, it seems that you need to communicate with the REST API on port 443